### PR TITLE
Allow for pipeline archives to be uploaded via CLI 

### PR DESCRIPTION
**Pull request by siganakis**
Allow for pipeline archives to be uploaded via the command line:
https://github.com/kubeflow/pipelines/issues/1998

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/pipelines/1999)

### CODE REVIEW

**Comment by siganakis** [comment]
@googlebot I signed it!

File: sdk/python/kfp/cli/pipeline.py [hunk 1]
```diff
@@ -0,0 +1,41 @@
+# Copyright 2018 Google LLC
```
Comment (hongye-sun) [inline sdk/python/kfp/cli/pipeline.py]: 2019 :P

File: sdk/python/kfp/cli/pipeline.py [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+def pipeline():
+    """Upload pipelines"""
+    pass
+
+@pipeline.command()
+@click.option('-f', '--package-file', type=click.Path(exists=True, dir_okay=False), help='Path of the pipeline package file.')
+@click.option('-n', '--pipeline-name', help='Name of the Pipeline.')
+@click.pass_context
+def upload(ctx, package_file, pipeline_name):
+    """list recent KFP runs"""
```
Comment (hongye-sun) [inline sdk/python/kfp/cli/pipeline.py]: probably need to change the comment.

File: sdk/python/kfp/cli/pipeline.py (hunk 1 above)
Comment (siganakis) [reply sdk/python/kfp/cli/pipeline.py]: Updated year to 2019  (in 3397efd)

File: sdk/python/kfp/cli/pipeline.py (hunk 2 above)
Comment (siganakis) [reply sdk/python/kfp/cli/pipeline.py]: Updated comment (in 3397efd)

**Pull request closed without merging**